src/ts/routes/authenticated/authenticated.tsx

```diff
@@ -3,20 +3,23 @@ import * as React from 'react'
 import { Redirect, RouteProps } from 'react-router'
 import { IRouteProps, RouteWithLayout } from '../withLayout'
 import { IAppState } from '../../background/store/all'
-import { INITIALIZE_ROUTE, UNLOCK_ROUTE } from '../../constants/routes'
+import { CREATE_PASSWORD_ROUTE, INITIALIZE_ROUTE, UNLOCK_ROUTE } from '../../constants/routes'
 
 const mapStateToProps = (state: IAppState) => {
   return {
     welcome: state.settings.welcome,
-    isLocked: state.account.locked
+    isLocked: state.account.locked,
+    accountCreated: state.account.created
   }
 }
 
 export class AuthenticatedRoute extends React.Component<StateProps & IRouteProps & RouteProps> {
   renderRoute (props: StateProps & IRouteProps & RouteProps) {
```

Comment:
probably it's worth doing 
`const { isLocked, welcome, accountCreated } = props` here
and then remove `props.` prefixes below ?

Reply:
upated